README.md

```diff
@@ -14,55 +14,73 @@
 
 # Aragon OSx Protocol
 
-The Aragon OSx protocol is the foundation layer of the new Aragon stack. It allows users to create, manage, and customize DAOs in a way that is lean, adaptable, and secure.
+The Aragon OSx protocol is the foundation layer of the new Aragon stack. It
```

Comment:
What has caused this change in the context of this PR? If prettier, why not earlier?
IMO, the `.md` got less readable now

Reply:
Strange, But i review this file in the branch it looks fine,
why here this is happening here?

Reply:
this is my default formatter that I use. Reverted this change